public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -312,10 +312,6 @@ export function requestSupportsPartitioning(queries: LokiQuery[]) {
     return false;
   }
 
-  if (isLogsQuery(queries[0].expr)) {
```

Comment:
Just double checking that with this PR you will be enabling logs chunking. If you meant to make this change, 👍

Reply:
yes, thanks for checking, it is intentional. 
i understand the logs-visualization is not perfect yet regarding this, but it is good enough for a first iteration based on my tests (i mean, it does not crash 😄 ).